axiom/monitors_integration_test.go

```diff
@@ -137,4 +139,35 @@ func (s *MonitorsTestSuite) TestCreateMatchMonitor() {
 	s.Require().NoError(err)
 	s.Require().NotNil(monitor)
 	s.Equal(axiom.MonitorTypeMatchEvent.String(), monitor.Type.String())
+	s.Equal(false, monitor.Disabled)
+
+	// Disable the match monitor
+	monitor.Disabled = true
+	updatedMonitor, err := s.client.Monitors.Update(s.ctx, monitor.ID, axiom.MonitorUpdateRequest{
+		Monitor: *monitor,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(updatedMonitor)
+	s.True(updatedMonitor.Disabled, "monitor should be disabled")
+
+	// Verify the monitor is disabled
+	monitor, err = s.client.Monitors.Get(s.ctx, monitor.ID)
+	s.Require().NoError(err)
+	s.Require().NotNil(monitor)
+	s.True(monitor.Disabled, "monitor should be disabled")
+
+	// Re-enable the match monitor
+	monitor.Disabled = false
+	updatedMonitor, err = s.client.Monitors.Update(s.ctx, monitor.ID, axiom.MonitorUpdateRequest{
+		Monitor: *monitor,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(updatedMonitor)
+	s.False(updatedMonitor.Disabled, "monitor should be enabled")
+
+	// Verify the monitor is re-enabled
+	monitor, err = s.client.Monitors.Get(s.ctx, monitor.ID)
+	s.Require().NoError(err)
+	s.Require().NotNil(monitor)
+	s.False(monitor.Disabled, "monitor should be enabled")
```

Comment:
In general +1 for additional testing but how do those tests reflect the changes being made?

Reply:
I'm testing the disabled functionality which is the first I'm adding, it's why it's working on development and not staging integration tests